odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1671,76 +1673,79 @@ def _validate_module_views(self, module):
 
 
 class ResetViewArchWizard(models.TransientModel):
-    """ A wizard to reset views architecture. """
+    """ A wizard to compare and reset views architecture. """
     _name = "reset.view.arch.wizard"
     _description = "Reset View Architecture Wizard"
 
-    def _default_view_id(self):
-        view_id = self._context.get('active_model') == 'ir.ui.view' and self._context.get('active_id') or []
-        return view_id
-
-    view_id = fields.Many2one('ir.ui.view', string='View', default=_default_view_id)
+    view_id = fields.Many2one('ir.ui.view', string='View')
     view_name = fields.Char(related='view_id.name', string='View Name')
-    arch_diff = fields.Html(string='Architecture Diff', compute='_compute_arch_diff', readonly=True, sanitize_tags=False)
+    arch_diff = fields.Html(string='Architecture Diff', readonly=True,
+                            compute='_compute_arch_diff', sanitize_tags=False)
     reset_mode = fields.Selection([
         ('soft', 'Restore previous version (soft reset).'),
-        ('hard', 'Reset to file version (hard reset).')
-    ], string='Reset Mode', default='soft', required=True, help="You might want to try a soft reset first.")
+        ('hard', 'Reset to file version (hard reset).'),
+        ('other_view', 'Reset to another view.')],
+        string='Reset Mode', default='soft', required=True,
+        help="You might want to try a soft reset first.")
+    view_to_id = fields.Many2one('ir.ui.view', string='Compare To View')
```

Comment:
name of var

Reply:
Spent too much time on this one, changed it from `view_to_compare_to_id`, `compare_view_id`, didn't find anything good.
I'll use `compare_view_id` again let me know if you have a better idea.